opensrs/opensrsapi.py

```diff
@@ -651,3 +651,21 @@ def _make_registrant_verification_call(self, domain_name, operation):
             object='domain',
             attributes={'domain': domain_name}
         ).get_data()
+
+    def enable_auto_renew(self, cookie, domain):
```

Comment:
Yeah :( That's sad...  So, we should choose the one which suits better to what we pass.  Personally, I prefer `domain_name` since in this context `domain` sounds like a resource we operate with.